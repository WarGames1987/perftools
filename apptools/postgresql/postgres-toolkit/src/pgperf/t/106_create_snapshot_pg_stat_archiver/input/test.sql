SELECT pgperf.create_snapshot_pg_stat_archiver(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;

SELECT pgperf.create_snapshot_pg_stat_archiver(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;

SELECT pgperf.create_snapshot_pg_stat_archiver(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;

SELECT pgperf.delete_snapshot_pg_stat_archiver(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;

SELECT pgperf.delete_snapshot_pg_stat_archiver(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;

SELECT pgperf.delete_snapshot_pg_stat_archiver(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
