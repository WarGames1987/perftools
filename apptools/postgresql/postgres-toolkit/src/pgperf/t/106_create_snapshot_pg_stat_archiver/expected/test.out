SELECT pgperf.create_snapshot_pg_stat_archiver(0);
 create_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

SELECT pgperf.create_snapshot_pg_stat_archiver(1);
 create_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

SELECT pgperf.create_snapshot_pg_stat_archiver(2);
 create_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_archiver(1);
 delete_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_archiver(2);
 delete_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_archiver(0);
 delete_snapshot_pg_stat_archiver 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_bgwriter;
 count 
-------
     0
(1 row)

